### ISSUE TITLE
Leaking sensitive information?

### ISSUE THREAD

**Issue by benderunit** [Dindaleon/hapi-react-starter-kit#23]
Is this http://hapi-reactstarterkit.rhcloud.com/ an example for running this starter kit in an production environment? If it is, I think it's leaking sensitive informations through the running webpack server. It serves informations like the iron secret through the config.js file.

**Comment by Dindaleon** [comment]
@benderunit Yes indeed. What do you suggest? Perhaps, having the secret key as environment variable?

**Comment by nym** [comment]
Does it exhibit the same problem when running as prod?

**Comment by FullStackForger** [comment]
No it does not @nym 
You can run it

```
SECRET="YOUR_SECRET_KEY" node server.js
```

with key set to your environment. This is how it is done to protect code from commiting sensitive information into repositories.
